doc/Command-Reference.md

```diff
@@ -1730,6 +1734,87 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#bgp)
 
+## Container Auto-restart
+SONiC includes a feature in which Docker containers can be automatically shut
+down and restarted if one of critical processes running in the container exits
+unexpectedly. Restarting the entire container ensures that configureation is 
+reloaded and all processes in the container get restarted, thus increasing the
+likelihood of entering a healthy state.
+
+### Container Auto-restart show commands
+
+**show container feature autorestart**
+
+This command will display the status of auto-restart feature for containers.
+
+- Usage:
+  ```
+  show container feature autorestart [<container_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container
+  ```
+
+  ```
+  admin@sonic:~$ show container feature
+  ```
```

Comment:
Unnecessary, as these are not complete commands. Please remove.

Reply:
Removed.